crates/ty_python_semantic/src/types.rs

```diff
@@ -5559,6 +5637,7 @@ impl<'db> Type<'db> {
             | Self::LiteralString
             | Self::IntLiteral(_)
             | Self::BytesLiteral(_)
+            | Self::EnumLiteral(_)
```

Comment:
Ideally I guess we'd jump to the definition of the actual enum member here, right? But I know that this would be much harder. Worth a TODO?